input/tcplistener/tcplinelistener.go

```diff
@@ -76,7 +76,7 @@ func NewTCPLineListener(parentLogger logger.Logger, address string, testRecord f
 	}, boundAddr, nil
 }
 
-func (lsnr *tcpLineListener) Launch() {
```

Comment:
I would probably prefer to change all "lsnr" to "listener" for readability purposes...

Reply:
done